presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -193,8 +193,8 @@ def test_removed_pattern_recognizer_doesnt_work(self):
 
         # Make sure the analyzer doesn't get this entity
         recognizers_store_api_mock = RecognizerStoreApiMock()
-        analyze_engine = AnalyzerEngine(MockRecognizerRegistry(
-            recognizers_store_api_mock))
+        analyze_engine = AnalyzerEngine(registry= MockRecognizerRegistry(
+            recognizers_store_api_mock), nlp_engine=SpacyNlpEngine())
```

Comment:
I don't think we need actual Spacy for these tests. We can use a mock instead. https://github.com/microsoft/presidio/blob/74ea983cc50ff76d7fc734683ca897ff01ceb9ba/presidio-analyzer/tests/mocks/nlp_engine_mock.py